addons/mrp/models/mrp_workcenter.py

```diff
@@ -17,6 +17,9 @@
 from odoo.tools.float_utils import float_compare, float_round
 from odoo.tools.misc import get_lang
 
+# Used in Tests to reduce the amount of planning computation
+MAX_PLANNING_ITERATIONS = 50
```

Comment:
Not sure about adding constants for the sole meaning of being patched in a test.
If we want to do that, wouldn't it be more useful to set a system parameter instead? So that it can be overriden in a test, while also being a bit more configurable than a hard-limit?